remotespark/livyclientlib/livysession.py

```diff
@@ -12,7 +13,8 @@
 from .livyreliablehttpclient import LivyReliableHttpClient
 
 
-class LivySession(object):
+
+class LivySession(GuidMixin, object):
```

Comment:
Mohamed, the `__init__` method does not get called automatically.

hdiuser@hn0-jupyte:~$ python
Python 2.7.6 (default, Jun 22 2015, 17:58:13)
[GCC 4.8.2] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> class A(object):
...     def __init__(self):
...             print 'yo'
...
>>> class B(object):
...     def __init__(self):
...             print 'wat'
...
>>> b = B()
wat
>>>